src/ws/hooks/item.ts

```diff
@@ -320,4 +357,142 @@ export const configureWsItemHooks = (websocketClient: WebsocketClient) => ({
       };
     }, [userId]);
   },
+
+  useRecycledItemsUpdates: (userId?: UUID | null) => {
+    const queryClient = useQueryClient();
+    useEffect(() => {
+      if (!userId) {
+        return () => {
+          // do nothing
+        };
+      }
+
+      const channel: Channel = { name: userId, topic: TOPICS.ITEM_MEMBER };
+
+      const handler = (event: ItemEvent) => {
+        if (event.kind === KINDS.RECYCLEBIN) {
+          const current =
+            queryClient.getQueryData<List<ItemRecord>>(RECYCLED_ITEMS_KEY);
+
+          if (current) {
+            const item: ItemRecord = convertJs(parseStringToDate(event.item));
+            let mutation;
+
+            switch (event.op) {
+              case OPS.CREATE: {
+                if (!current.find((i) => i.id === item.id)) {
+                  mutation = current.push(item);
+                  queryClient.setQueryData(RECYCLED_ITEMS_KEY, mutation);
+                  queryClient.setQueryData(buildItemKey(item.id), item);
```

Comment:
Not sure here. I this called when an item is recycled i.e. "created" in the bin ? I guess the key should be specific to the recycled items.